main.py

```diff
@@ -46,6 +71,26 @@ def submit(driver, id_='driving-licence-submit'):
 
 
 def notify(msg):
+    msg += '\nHead over to https://driverpracticaltest.dvsa.gov.uk/ to book now'
+    if ENABLE_TELEGRAM:
+        send_text = f'https://api.telegram.org/bot{TELEGRAM_BOT_TOKEN}/sendMessage?chat_id={TELEGRAM_CHAT_ID}&parse_mode=Markdown&text=' + msg
+        requests.get(send_text)
+    if ENABLE_EMAIL:
+        message={}
```

Comment:
This line seems redundant?

Reply:
needed to send notifs, updated in next commit